.vscode/c_cpp_properties.json

```diff
@@ -1,17 +1,17 @@
-{
-  "configurations": [
-    {
-      "browse": {
-        "databaseFilename": "",
-        "limitSymbolsToIncludedHeaders": true
-      },
-      "includePath": [
-        "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/include/**",
-        "/opt/ros/melodic/include/**",
-        "/usr/include/**"
-      ],
-      "name": "ROS"
-    }
-  ],
-  "version": 4
+{
+  "configurations": [
+    {
+      "browse": {
+        "databaseFilename": "",
+        "limitSymbolsToIncludedHeaders": true
+      },
+      "includePath": [
+        "/home/alexemdesgagne/Workspaces/ros_sonia_ws/devel/include/**",

```

Comment:
This will make an error when trying to debug on an other computer than Alex's

Reply:
Technically, you can't debug matlab code on vs code. so git ignore ?